tests/shardddl1/run.sh

```diff
@@ -413,6 +413,11 @@ function DM_RestartMaster() {
 function restart_master_on_pos() {
 	if [ "$1" = "$2" ]; then
 		restart_master
+		# because we disabled auto-resume feature to satisfy other tests, we manually resume once to handle task was paused
```

Comment:
maybe we can add `rsume-task` to func `restart_master` or add a new func `restart_master_and_resume_task`?

this might avoid duplicate fixes😬

Reply:
some tests rely on auto-resume but some not, so we might check and split them, and these manually resuming can be removed.

cases using `restart_master` may not expected resuming, and `restart_master_and_resume_task` requires too much inspection, not worthy to me.